### ISSUE TITLE
Using externals results in sourcemap misalignment

### ISSUE THREAD

**Issue by ldstein** [webpack/webpack#4785]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Sourcemaps in Chrome debugger are out of alignment when Webpack config includes both `devtool` and `externals`.
```js
// webpack.config.js
{
   entry:'./index.js',
   devtool: 'cheap-module-source-map'
   externals:{someUtil:'someUtil'},
   ...
}
```

**If the current behavior is a bug, please provide the steps to reproduce.**
See https://github.com/ldstein/webpack-sourcemap-test for demonstration of issue.

**What is the expected behavior?**
Breakpoints should match line numbers in Chrome Debugger.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**
- webpack 2.4.1
- webpack-dev-server 2.4.4
- MS Windows 10 Pro 64bit
- Chrome 57.0.2987.133 64bit

**Comment by ldstein** [comment]
Have tested further - this issue occurs specifically when webpack config has a combination of `devtool:cheap-module-source-map` and `externals`

I have updated https://github.com/ldstein/webpack-sourcemap-test to better demonstrate the issue:

- Generate two outputs from the same index.js file.
- Output 1 has a single module configured as an external.
- Output 2 has no externals configured.
- Both outputs have devtool set to `cheap-module-source-map`
- Both outputs generate identical bundle js files

When diffing the sourcemaps, there is a single character different which is causing the misalignment problem in Chrome debugger:

![Screenshot of Debugger](https://raw.githubusercontent.com/ldstein/webpack-sourcemap-test/master/doc/diff.png)

**Comment by sokra** [comment]
fixed in `source-list-map@1.1.2`. Upgrade.
